.github/dependabot.yml

```diff
@@ -42,7 +42,10 @@ updates:
     ignore:
       # TODO: https://github.com/dequelabs/axe-core-maven-html/issues/378
       - dependency-name: "org.seleniumhq.selenium:selenium-java"
-        versions: [">=4.14.0"]
+        versions: [ ">=4.14.0" ]
+      # TODO: https://github.com/dequelabs/axe-core-maven-html/issues/393
+      - dependency-name: "org.mockito:mockito-core"
+          versions: [ ">=5.0.0" ]
```

Comment:
This is outside the scope of the pr. Would be nice to remove and put in it's own pr.

Reply:
Whoops my mistake - This can be removed along with the corresponding issue. I did not need need Mockito to mock the driver, just extending the base class and override to test functionality was sufficient!